test/utilities/apexUtils.ts

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright (c) 2023, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import {
+    TextEditor
+} from 'wdio-vscode-service';
+import * as utilities from '../utilities';
```

Comment:
I believe this is a circular reference.  apexUtils is inside utilities, but apexUtils is importing * from utilities (utilities/index.ts)...
which imports utilities/apexUtils...
which imports * from utilities...
which imports utilities/apexUtils...
.
.
.



;-)

Instead, you will need to import each dependency individually and replace
```
import * as utilities from '../utilities';
```

...with:
```
import {
    runCommandFromCommandPrompt
} from './commandPrompt';
import {
    pause
} from './miscellaneous';
```

...and then remove `utilities.` where `pause()` and `runCommandFromCommandPrompt()` are called.